scripts/utils/update_package.py

```diff
@@ -119,7 +119,8 @@ def update_dependencies(package):
                     content,
                 )
                 updates = True
-                if dependency == package[:-3]:  # Metapackage
+                # both should be all lowercase via the linter, but let's be sure here
+                if dependency.lower() == package[:-3].lower():  # Metapackage
```

Comment:
This shouldn't be needed as the linter is now checking this, but it should be more robust 👍